code/game/machinery/buttons.dm

```diff
@@ -31,7 +31,9 @@
 	var/cooldown = 1 SECOND
 
 /obj/machinery/button/buildable
-	uncreated_component_parts = null
+	uncreated_component_parts = list(
+		/obj/item/stock_parts/radio/transmitter/basic = 1,
```

Comment:
Still seeing things with noninteractive radio components.

Reply:
Well, actually that was supposed to be an apc component, so I fixed that XD